worlds/tits_the_3rd/locations.py

```diff
@@ -22,6 +23,8 @@ def create_location(multiworld: MultiWorld, player: int, region_name: str, locat
     """
     region = multiworld.get_region(region_name, player)
     location = TitsThe3rdLocation(player, location_name, location_table[location_name], region)
+    if location.address == location_table[LocationName.chapter1_boss_defeated]:
+        location.place_locked_item(TitsThe3rdItem("Bennu Defeated (Victory)", ItemClassification.progression, 500000, player))
```

Comment:
Should we add this as an item name / actual item?

Reply:
This is purely for the alpha test, it will get changed to an actual item later when we add area progression